diff -Nur suricata-4.1.8.orig/suricata.yaml.in suricata-4.1.8/suricata.yaml.in
--- suricata-4.1.8.orig/suricata.yaml.in	2020-04-27 20:59:48.000000000 +0200
+++ suricata-4.1.8/suricata.yaml.in	2020-09-21 12:07:29.396077128 +0200
@@ -12,7 +12,8 @@
 vars:
   # more specific is better for alert accuracy and performance
   address-groups:
-    HOME_NET: "[192.168.0.0/16,10.0.0.0/8,172.16.0.0/12]"
+    HOME_NET: "[192.168.1.0/24]"
+    #HOME_NET: "[192.168.0.0/16,10.0.0.0/8,172.16.0.0/12]"
     #HOME_NET: "[192.168.0.0/16]"
     #HOME_NET: "[10.0.0.0/8]"
     #HOME_NET: "[172.16.0.0/12]"
@@ -1129,6 +1130,7 @@
 # Magic file. The extension .mgc is added to the value here.
 #magic-file: /usr/share/file/magic
 @e_magic_file_comment@magic-file: @e_magic_file@
+magic-file: /usr/share/misc/magic
 
 # GeoIP2 database file. Specify path and filename of GeoIP2 database
 # if using rules with "geoip" rule option.
@@ -1881,51 +1883,51 @@
 @has_suricata_update_comment@default-rule-path: @e_defaultruledir@
 
 @has_suricata_update_comment@rule-files:
-@has_suricata_update_comment@ - botcc.rules
+@has_suricata_update_comment@# - botcc.rules
 @has_suricata_update_comment@ # - botcc.portgrouped.rules
-@has_suricata_update_comment@ - ciarmy.rules
-@has_suricata_update_comment@ - compromised.rules
-@has_suricata_update_comment@ - drop.rules
-@has_suricata_update_comment@ - dshield.rules
+@has_suricata_update_comment@# - ciarmy.rules
+@has_suricata_update_comment@# - compromised.rules
+@has_suricata_update_comment@# - drop.rules
+@has_suricata_update_comment@# - dshield.rules
 @has_suricata_update_comment@# - emerging-activex.rules
-@has_suricata_update_comment@ - emerging-attack_response.rules
-@has_suricata_update_comment@ - emerging-chat.rules
-@has_suricata_update_comment@ - emerging-current_events.rules
-@has_suricata_update_comment@ - emerging-dns.rules
-@has_suricata_update_comment@ - emerging-dos.rules
-@has_suricata_update_comment@ - emerging-exploit.rules
-@has_suricata_update_comment@ - emerging-ftp.rules
+@has_suricata_update_comment@# - emerging-attack_response.rules
+@has_suricata_update_comment@# - emerging-chat.rules
+@has_suricata_update_comment@# - emerging-current_events.rules
+@has_suricata_update_comment@# - emerging-dns.rules
+@has_suricata_update_comment@# - emerging-dos.rules
+@has_suricata_update_comment@# - emerging-exploit.rules
+@has_suricata_update_comment@# - emerging-ftp.rules
 @has_suricata_update_comment@# - emerging-games.rules
 @has_suricata_update_comment@# - emerging-icmp_info.rules
 @has_suricata_update_comment@# - emerging-icmp.rules
-@has_suricata_update_comment@ - emerging-imap.rules
+@has_suricata_update_comment@# - emerging-imap.rules
 @has_suricata_update_comment@# - emerging-inappropriate.rules
 @has_suricata_update_comment@# - emerging-info.rules
-@has_suricata_update_comment@ - emerging-malware.rules
-@has_suricata_update_comment@ - emerging-misc.rules
-@has_suricata_update_comment@ - emerging-mobile_malware.rules
-@has_suricata_update_comment@ - emerging-netbios.rules
-@has_suricata_update_comment@ - emerging-p2p.rules
-@has_suricata_update_comment@ - emerging-policy.rules
-@has_suricata_update_comment@ - emerging-pop3.rules
-@has_suricata_update_comment@ - emerging-rpc.rules
+@has_suricata_update_comment@# - emerging-malware.rules
+@has_suricata_update_comment@# - emerging-misc.rules
+@has_suricata_update_comment@# - emerging-mobile_malware.rules
+@has_suricata_update_comment@# - emerging-netbios.rules
+@has_suricata_update_comment@# - emerging-p2p.rules
+@has_suricata_update_comment@# - emerging-policy.rules
+@has_suricata_update_comment@# - emerging-pop3.rules
+@has_suricata_update_comment@# - emerging-rpc.rules
 @has_suricata_update_comment@# - emerging-scada.rules
 @has_suricata_update_comment@# - emerging-scada_special.rules
-@has_suricata_update_comment@ - emerging-scan.rules
+@has_suricata_update_comment@# - emerging-scan.rules
 @has_suricata_update_comment@# - emerging-shellcode.rules
-@has_suricata_update_comment@ - emerging-smtp.rules
-@has_suricata_update_comment@ - emerging-snmp.rules
-@has_suricata_update_comment@ - emerging-sql.rules
-@has_suricata_update_comment@ - emerging-telnet.rules
-@has_suricata_update_comment@ - emerging-tftp.rules
-@has_suricata_update_comment@ - emerging-trojan.rules
-@has_suricata_update_comment@ - emerging-user_agents.rules
-@has_suricata_update_comment@ - emerging-voip.rules
-@has_suricata_update_comment@ - emerging-web_client.rules
-@has_suricata_update_comment@ - emerging-web_server.rules
+@has_suricata_update_comment@# - emerging-smtp.rules
+@has_suricata_update_comment@# - emerging-snmp.rules
+@has_suricata_update_comment@# - emerging-sql.rules
+@has_suricata_update_comment@# - emerging-telnet.rules
+@has_suricata_update_comment@# - emerging-tftp.rules
+@has_suricata_update_comment@# - emerging-trojan.rules
+@has_suricata_update_comment@# - emerging-user_agents.rules
+@has_suricata_update_comment@# - emerging-voip.rules
+@has_suricata_update_comment@# - emerging-web_client.rules
+@has_suricata_update_comment@# - emerging-web_server.rules
 @has_suricata_update_comment@# - emerging-web_specific_apps.rules
-@has_suricata_update_comment@ - emerging-worm.rules
-@has_suricata_update_comment@ - tor.rules
+@has_suricata_update_comment@# - emerging-worm.rules
+@has_suricata_update_comment@# - tor.rules
 @has_suricata_update_comment@# - decoder-events.rules # available in suricata sources under rules dir
 @has_suricata_update_comment@# - stream-events.rules  # available in suricata sources under rules dir
 @has_suricata_update_comment@ - http-events.rules    # available in suricata sources under rules dir
